src/Microsoft.Identity.Client/Http/HttpResponse.cs

```diff
@@ -33,7 +33,7 @@
 
 namespace Microsoft.Identity.Client.Http
 {
-    internal class HttpResponse : IHttpWebResponse
+    internal class HttpResponse
```

Comment:
nit: can you clean up formatting in this class while you're there?  { on same line, extra blank lines on 62, etc)